pkg/scheduler/ugm/manager.go

```diff
@@ -136,6 +139,8 @@ func (m *Manager) DecreaseTrackedResource(queuePath, applicationID string, usage
 	if !decreased {
 		return decreased
 	}
+
+	// remove user after getting group tracker, or ensureGroup may function incorrectly
```

Comment:
modify this comment accordingly or you can remove the comment

Reply:
Updated it. Thanks.